main.js

```diff
@@ -0,0 +1,84 @@
+const navtoggle = document.querySelector('#navToggle');
+const nav = document.querySelector('nav');
+const navIcon = document.querySelectorAll('.navIcon');
+const hamburger = document.querySelector('#hamburger');
+
+navtoggle.addEventListener('click', () => {
+  nav.classList.toggle('open');
+  navIcon.forEach((icon) => {
+    icon.classList.toggle('hidden');
+  });
+});
+
+window.addEventListener('resize', () => {
+  if (document.body.clientWidth > 768) {
+    nav.classList.remove('open');
+    navIcon.forEach((icon) => {
+      icon.classList.add('hidden');
+    });
+    hamburger.classList.remove('hidden');
+  }
+});
+
+const speakerList = [{
+  imageLink: './components/peak.png',
+  title: 'Robert Pong',
+  description: 'Lorem ipsum dolor.',
+  fullDescription: ' Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+},
+
+{
+  imageLink: './components/peak.png',
+  title: 'Denis Lapo',
+  description: 'Lorem ipsum dolor.',
+  fullDescription: ' Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+},
+
+{
+  imageLink: './components/speaker4.jpg',
+  title: 'Dan Levis',
+  description: 'Lorem ipsum dolor.',
+  fullDescription: ' Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+},
+
+{
+  imageLink: './components/speaker1.jpg',
+  title: 'Alicia Keys',
+  description: 'Lorem ipsum dolor.',
+  fullDescription: ' Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+},
+
+{
+  imageLink: './components/speaker2.jpg',
+  title: 'John Don',
+  description: 'Lorem ipsum dolor.',
+  fullDescription: ' Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+},
+
+{
+  imageLink: './components/speaker3.jpg',
+  title: 'Kevin Brown',
+  description: 'Lorem ipsum dolor.',
+  fullDescription: ' Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+}];
+
+const cardContainer = document.querySelector('.featured-speakers');
+
+const speakers = () => {
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < speakerList.length; i++) {
+    cardContainer.innerHTML += `<div class="speakers">
+      <img id="speaker1" class="speak-img" src="${speakerList[i].imageLink}" alt="${speakerList[i].title}">
+      <div class="speak-element">
+        <h3 id="title">${speakerList[i].title}</h3>
+        <h6 id="subTitle">${speakerList[i].description}
+        </h6>
+        <p id="paragraph">
+          ${speakerList[i].fullDescription}
+        </p>
```

Comment:
Please, endeavor to use the background behind the image as given in the design. Sticking to a given design is a good skill to have now because in a typical workplace (which you're being prepared for at Microverse), you might be required to stick to a given design to the later.

Your design:
<img width="317" alt="image" src="https://user-images.githubusercontent.com/20554583/197339476-35694f4b-86f0-413f-b170-c1676f724c06.png">

Figma design:
<img width="223" alt="image" src="https://user-images.githubusercontent.com/20554583/197339424-eda63ac8-7fc8-4541-8f45-b8f1e2845891.png">